jwt/api.py

```diff
@@ -110,14 +129,16 @@ def encode(self, payload, key, algorithm='HS256', headers=None, json_encoder=Non
 
         return b'.'.join(segments)
 
-    def decode(self, jwt, key='', verify=True, algorithms=None, **kwargs):
+    def decode(self, jwt, key='', verify=True, algorithms=None, options=None, **kwargs):
         payload, signing_input, header, signature = self._load(jwt)
 
         if verify:
-            self._verify_signature(payload, signing_input, header, signature,
-                                   key, algorithms)
+            merged_options = self._merge_options(override_options=options)
+            if merged_options.get('verify_signature'):
+                self._verify_signature(payload, signing_input, header, signature,
+                                       key, algorithms)
 
-            self._validate_claims(payload, **kwargs)
+            self._validate_claims(payload, options=merged_options, **kwargs)
```

Comment:
I agree with you that people screw things up because they are hard and complicated. However, I think that's why its simplest to present a secure set of defaults and make people aware of the fact that they can modify them at their own risk.

@michaeldavis-wf You might want to add to the README that changing the defaults is a behavior done at their own risk and may make their application less secure.